tests/integration/steps_multi_consumer_double_sign.go

```diff
@@ -0,0 +1,224 @@
+package main
+
+// simulates double signing on provider and vsc propagation to consumer chains
+//
+// Note: These steps would be affected by slash packet throttling, since the
+// consumer-initiated slash steps are executed after consumer-initiated downtime
+// slashes have already occurred. However slash packet throttling is
+// psuedo-disabled in this test by setting the slash meter replenish
+// fraction to 1.0 in the config file.
+//
+// only double sign on provider chain will cause slashing and tombstoning
+func stepsMultiConsumerDoubleSign(consumer1, consumer2 string) []Step {
```

Comment:
This is just copy/pasted from the old func with no changes, correct?

Reply:
Yes